node/libs/roles/src/validator/messages/block.rs

```diff
@@ -4,6 +4,56 @@ use super::CommitQC;
 use crypto::{sha256, ByteFmt, Text, TextFmt};
 use std::fmt;
 
+/// Version of the consensus algorithm used by the leader in a given view.
+/// Currently it is stored in the BlockHeader (TODO: consider whether it should be present in every
+/// consensus message).
```

Comment:
The argument in favor is that if we have a new version which changes the semantics of the consensus message it may happen that the validator running old binary will be incorrectly interpreted by a validator running a new binary.

Reply:
I think we should do it then.